### ISSUE TITLE
Running ms2query 0.3.0 leads to error

### ISSUE THREAD

**Issue by sebabuel** [iomega/ms2query#132]
Running ms2query in version 0.3.0 with the provided [script](https://github.com/iomega/ms2query#run-ms2query) leads to the following error:

```powershell
ms2library = create_library_object_from_one_dir(ms2query_library_files_directory, default_library_file_base_names())
  File "/usr/local/lib/python3.8/site-packages/ms2query/ms2library.py", line 485, in create_library_object_from_one_dir
    return MS2Library(sqlite_file_name, s2v_model_file_name, ms2ds_model_file_name, s2v_embeddings_file_name,
  File "/usr/local/lib/python3.8/site-packages/ms2query/ms2library.py", line 90, in __init__
    self.ms2query_model = load_ms2query_model(ms2query_model_file_name)
  File "/usr/local/lib/python3.8/site-packages/ms2query/utils.py", line 30, in load_ms2query_model
    raise ValueError("The MS2Query model file is expected to end on .pickle")
ValueError: The MS2Query model file is expected to end on .pickle
```

I did not have this error with the previous ms2query version, so maybe the download in the script might be linking to the wrong model files?!

**Comment by florian-huber** [comment]
@niekdejonge the zenodo link is indeed still the old one (https://zenodo.org/record/5645246/). Are the current files already somewhere on zenodo so that we can update the link?

-->
 https://github.com/iomega/ms2query/blob/4de961dcd1938749aae31c2dd0bf4d8315053923/ms2query/run_ms2query.py#L36

**Comment by niekdejonge** [comment]
Hi Sebastian, sorry for the late reply. I missed your comment. If you want a quick reply, you can also tag me, since if you tag me I also get an email, so I might see it quicker. 
It is indeed a problem with the old files, we switched from neural networks to random forest resulting in the file change. I was not aware that zenodo still contained an old model of a different file type. I will update them now (Hope I menage to finish today), so that should solve the issue, sorry for the inconvenience.

**Comment by niekdejonge** [comment]
Solved with #133
